public/js/project/utilities/math.js

```diff
@@ -68,6 +68,20 @@ module.exports = {
 		}
 		return validPoints;
 	},
+	getValidPreImageOval: function(matrix) {
```

Comment:
wouldn't hurt to do some validation on `matrix` here. 
especially if we start letting the user choose the matrix.